### ISSUE TITLE
Missing Text Messages?

### ISSUE THREAD

**Issue by yuuqilin** [MarvinXLII/BravelyCrowd#1]
## Issue Description

After unpacking, I'm unable to locate the text for difficulty adjustment options (Easy, Normal, Hard) in the file `romfs_unpacked\Common_en\MessageTable\crowd.xls`. These should be present in `MenuMessageData.btb`.

Modifying the text in cells Text Pntr 16272, 16282, and 16296 only changes the Easy, Normal, Hard text displayed in the first-level menu (see Image 1). However, the same text in the second-level menu remains unaffected (see Image 2).

I've searched through `crowd.xls` files in other folders but haven't found any other locations storing these three text strings.

## Additional Information

- Image 1: ![Second-level menu with unmodified text](https://github.com/user-attachments/assets/e1dec0ec-bf7d-46ee-a885-5565d2ec160f)
- Image 2: ![First-level menu with modified text](https://github.com/user-attachments/assets/6b2892a9-4765-4589-bba1-20a0d1ad69fc)

## Questions

1. Are these text strings stored elsewhere?
2. Is there a different method to modify the second-level menu text?

Any insights or suggestions would be greatly appreciated. Thank you!

**Comment by MarvinXLII** [comment]
Unfortunately some of the UI text is stored separately, and those files are much more complicated to parse into and build from any kind of spreadsheet. The best you could do is edit the text in a hex editor, but you would be stuck with the same word lengths.

For reference, the file is Graphics/UI_en/Layout/15_Config starting at byte 0x6944.

**Comment by yuuqilin** [comment]
Thank you for your response. I've attempted to modify the 15_Config file as you suggested. Since I'm uncertain about the specific function of each 16-bit block within the file, I've decided to relocate the first difficulty option, "Easy," to the third position. Additionally, I've modified the "txt_msg_" string and adjusted the values at addresses 0x68FA, 0x697E, and 0x69FE to reflect this change. These addresses seem to control the display order of the options in the menu. The final result can be seen in the attached image:

![Airy Difficulty](https://github.com/user-attachments/assets/f0167e3a-7af2-4dc0-9245-208f876961a5)

I've tested the modified file using Citra, and the emulator doesn't crash. The difficulty options can be switched correctly, as expected.
